terraform/sqs.tf.json.template.py

```diff
@@ -8,35 +8,36 @@
         "resource": [
             {
                 "aws_sqs_queue": {
-                    "notifications_queue": {
+                    config.unqual_notifications_queue_name(): {
```

Comment:
This is great. I wanted to do something about this, but I didn't want to step too far out of scope.